src/tests/cucumber/requirements.txt

```diff
@@ -1,2 +1,3 @@
 behave
-pyyaml
\ No newline at end of file
+pyyaml
+testresources
```

Comment:
why do you need this new requirement ?

Reply:
Without it I had an error installing requirements saying it was missing

Reply:
I have removed the addition.
Locally I uninstalled testresources and it works properly